packages/pocket-ic/src/lib.rs

```diff
@@ -869,6 +869,13 @@ impl PocketIc {
         runtime.block_on(async { self.pocket_ic.get_subnet(canister_id).await })
     }
 
+    /// Returns subnet metrics for a given subnet.
+    /// Panics if the subnet does not exist.
```

Comment:
Have you considered returning an error or an option instead?

Reply:
Done.